test/integration/features/cmd-addons.feature

```diff
@@ -8,12 +8,9 @@ Feature: Addons command and its subcommands
       And executing "minishift add-ons" fails
       And executing "minishift add-on" fails
 
- Scenario: Installing default add-ons manually
-     When executing "minishift addons install --defaults" succeeds
-     Then stdout should contain
-      """
-      Default add-ons anyuid, admin-user, xpaas, registry-route installed
-      """
+  @minishift-only
```

Comment:
I am intrigued, was this working for CDK before?

Reply:
Unrelated, line 8 - 9:

      And executing "minishift add-ons" fails
      And executing "minishift add-on" fails

I think this should be removed. Why would we want to ensure that this fails?

Reply:
Personally I would try to keep the use of '@minishift-only' only minimal. I would use liberal regular expressions for the default add-ons which are enabled per default in the CDK. This way the whole test can run against Minishift or CDK.

Reply:
@hferentschik I just reverting the changes I had done as part of https://github.com/minishift/minishift/pull/1419/. I think we can remove the @minishift-only from here and it would work fine in CDK . @agajdosi @praveenkumar WDYT?

Reply:
@LalatenduMohanty If we remove it, then it will fail, as `minishift addons list` will return addons which are enabled by default on CDK.

@hferentschik I am trying to keep the use of `minishift-only` minimal.  Here it makes sense I think as we also want to test that default addons are installed and disabled.

Reply:
@hferentschik @praveenkumar @agajdosi I kept the the test @minishift-only as we should also check that add-ons are disabled by default for upstream Minishift.